components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -182,15 +215,39 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
             }
           });
       });
+
+    combineLatest([
```

Comment:
I think this whole block could use an explanatory comment, especially regarding how we're using `teamProjectsLeftToFetch`